### DISCUSSION TITLE
We are unable to complete your signup. Please read this guide for further details.

### DISCUSSION THREAD

**Discussion by ItzCrazyKns** [vercel/community#3730] [Account Issue] [answered]
### Question

Hey, I've created a new account with email `kushagra20103@gmail.com` but when i try to login it says `We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.` Please help.

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Providing additional information about your account and the project you are deploying can help accelerate the account verification process.

**Reply by Aminul721** [reply]
I'm already mail to this mail "registration@vercel.com", and I said my problem, but now I'm what can I do

Thanks !!

**Comment by Aminul721** [comment]
@ItzCrazyKns Hey, I'm facing same problem, your vercel account login issues done?
